src/domain/schemas/document-schemas.js

```diff
@@ -32,7 +32,7 @@ export const createRevisionBodySchema = joi.object({
   language: joi.string().case('lower').required(),
   sections: joi.array().items(sectionSchema).required(),
   appendTo: documentRevisionAppendToSchema.optional(),
-  tags: joi.array(),
+  tags: joi.array().items(joi.string()).required(),
```

Comment:
:+1:

Reply:
Cool, we can also put a min(3) and max(30) so it matches more closely to the FE validation on tags 
.items(joi.string().min(3).max(30))

Reply:
To me client/server side validation should insure **business** validation, the store should insure **data/model consistency**. For this reason I would not add this extra restriction.

Reply:
Ok